src/Provider/GoogleMaps/GoogleMaps.php

```diff
@@ -274,12 +273,13 @@ private function updateAddressComponent(AddressBuilder $builder, string $type, $
             case 'sublocality_level_3':
             case 'sublocality_level_4':
             case 'sublocality_level_5':
-                $newSubLocalityLevel = [
+                $subLocalityLevel = $builder->getValue('subLocalityLevel', []);
```

Comment:
Well, one of my first attempts was similar. but I used array_merge. ... You told me to remove array_merge, and I understood you wrongly and I tried to remove getValue(..). What a mess...